nautobot/docs/guides/graphql.md

```diff
@@ -72619,7 +72619,7 @@ gql = nb.graphql.query(query=query, variables=variables)
 print(json.dumps(gql.json, indent=2))
 ```
 
-In the updated script, we add the `variables = {"site_name": "ams"}` variable. We then update the query to let GraphQL know that we will be sending parameters to to filter by `site`. The updated output is still a JSON object. instead of fetching all devices, we are filtering by devices in the `ams` site. The `PyNautobot SDK` has some excellent GraphQL [examples](https://pynautobot.readthedocs.io/en/latest/api/core/graphql.html). Be sure to check out the documentation.
+In the updated script, we add the `variables = {"site_name": "ams"}` variable. We then update the query to let GraphQL know that we will be sending parameters to to filter by `site`. The updated output is still a JSON object. Instead of fetching all devices, we are filtering by devices in the `ams` site. The `PyNautobot SDK` has some excellent GraphQL [examples](https://pynautobot.readthedocs.io/en/latest/api/core/graphql.html). Be sure to check out the documentation.
```

Comment:
```suggestion
In the updated script, we add the `variables = {"site_name": "ams"}` variable. We then update the query to let GraphQL know that we will be sending parameters to to filter by `site`. The updated output is still a JSON object. Instead of fetching all devices, we are filtering by devices in the `ams` site. **PyNautobot** has some [excellent GraphQL examples](https://pynautobot.readthedocs.io/en/latest/api/core/graphql.html). Be sure to check out the documentation.
```

Reply:
Thanks for the suggestions. I think that I'm going to go back to screenshots that are collapsable. This should get the point across and reduce scroll hell. I'll get the changes in shortly!